constructs/rabbitmq.ts

```diff
@@ -0,0 +1,27 @@
+import { Construct } from 'constructs';
+import { Chart, Helm } from 'cdk8s';
+
+export class RabbitMQ extends Chart {
+  constructor(scope: Construct) {
+    super(scope, 'rabbitmq', {
+      labels: {
+        app: 'rabbitmq-master',
+      },
+    });
+
+    new Helm(this, 'rabbitmq', {
+      chart: 'bitnami/rabbitmq',
+      values: {
+        auth: {
+          user: 'user',
+          password: '3e3abae2-6325-11ec-90d6-0242ac120003',
+          erlangCookie: 'some random secret',
```

Comment:
I didn't actually mean this string...
not sure how spaces will be handled... (just generate a guid or something)